librz/main/rizin.c

```diff
@@ -184,18 +184,44 @@ static int main_help(int line) {
 			" user         %s %s (and %s)\n"
 			" file         ${filename}.rz\n"
 			"Plugins:\n"
-			" binrc        %s (elf, elf64, mach0, ..)\n"
-			" RZ_USER_PLUGINS %s\n"
-			" RZ_LIB_PLUGINS %s\n"
+			" binrc            %s (elf, elf64, mach0, ..)\n"
+			" RZ_USER_PLUGINS  %s\n"
+			" RZ_LIB_PLUGINS   %s\n"
 			" RZ_EXTRA_PLUGINS %s\n"
 			"Environment:\n"
-			" RZ_DEBUG      if defined, show error messages and crash signal\n"
-			" RZ_DEBUG_ASSERT=1 set a breakpoint when hitting an assert\n"
-			" RZ_MAGICPATH %s\n"
-			" RZ_NOPLUGINS do not load rizin shared plugins\n"
-			" RZ_RCFILE    %s (user preferences, batch script)\n"
-			" RZ_DATAHOME %s\n"
-			" RZ_VERSION   contains the current version of rizin\n"
+			" _NT_DEBUGGER_EXTENSION_PATH  path(s) to debugger extension DLLs\n"
+			" ALACRITTY_LOG                alacritty log file's path\n"
+			" ANSICON                      ansicon's W & H of the buffer and w & h of the window in the form of: \"WxH (wxh)\"\n"
+			" CC                           compiler's name or path\n"
+			" DEBUGINFOD_URLS              e bin.dbginfo.debuginfod_urls - use alternative debuginfod server\n"
+			" COLORTERM                    extra color capabilities for a terminal\n"
+			" COLUMNS                      terminal columns to use\n"
+			" LD_LIBRARY_PATH              path(s) to search for shared libraries at run time\n"
+			" LIBRARY_PATH                 path(s) to search for static and shared libraries at compile time\n"
+			" PATH                         path(s) with executables\n"
+			" RZ_CURL                      whether to use curl (for SSL support)\n"
+			" RZ_DEBUG                     if defined, show error messages and crash signal\n"
+			" RZ_DEBUG_ASSERT=1            set a breakpoint when hitting an assert\n"
+			" RZ_DEBUG_TOOL=gdb            debug tool to use when showing error messages and crash signal\n"
+			" RZ_DYLDCACHE_FILTER          dyld cache filter (MacOS dynamic libraries location(s) at runtime)\n"
+			" RZ_HTTP_AUTHFILE             HTTP Authentification user file\n"
+			" RZ_LOGCOLORS                 should the log output use colors (TODO)\n"
```

Comment:
Maybe remove since it's not implemented

Reply:
Log colors has been implemented by me https://github.com/rizinorg/rizin/blob/dev/librz/util/log.c#L33 (https://github.com/rizinorg/rizin/commit/3033f628657db5e6c15885746244bb16d95903f6#diff-883e79a66b0fd7d2f41f70f1e2a111c879a6e249c57609f8bc19187699547957R33)